docs/src/main/tut/contributing.md

```diff
@@ -0,0 +1,386 @@
+---
+layout: docs
```

Comment:
We could, however, it's not mandatory. If we want to move it to the root I'm fine with that. Nevertheless, I'd separate it from the current `CONTRIBUTING.md` file in the root. What do you think about `DEV_GUIDE.md` or something similar? (afterward, when it's copied to the microsite territory we can rename it on the fly).